bundles/model/org.eclipse.smarthome.model.core/src/main/java/org/eclipse/smarthome/model/core/internal/folder/FolderObserver.java

```diff
@@ -227,31 +219,39 @@ public boolean accept(File dir, String name) {
         }
     }
 
+    // TODO remove once #3562 got resolved
```

Comment:
I'd assume that you didn't want to revert this change with https://github.com/eclipse/smarthome/pull/4851/commits/ecb254c22bf15b00613f8f6c138a47047b815f88.